jobs/secureproxy/templates/config/nginx.conf.erb

```diff
@@ -153,35 +154,26 @@ http {
       ##
 
       access_by_lua_block {
-        local headers = ngx.req.get_headers()
+        local tic_secret = "<%= p('secureproxy.tic.secret') %>"
```

Comment:
If `secureproxy.tic.secret` is the default value of `""`, it looks like this would trust an empty `X-TIC-Secret`, since we check `tic_secret` against `nil` in the `tic` module. If that's correct, we could conditionally set `tic_secret` here:

```
<% if_p('secureproxy.tic.secret') do |secret| %>
local tic_secret = "<%= secret %>"
<% end %>
```

Or check against empty string and nil in the `tic` module.